src/main/java/org/gridsuite/study/server/RestResponseEntityExceptionHandler.java

```diff
@@ -59,6 +59,8 @@ protected ResponseEntity<Object> handleException(RuntimeException exception) {
                 case DELETE_EQUIPMENT_FAILED:
                 case DELETE_NODE_FAILED:
                 case DELETE_STUDY_FAILED:
+                case DELETE_MODIFICATIONS_FAILED:
+                case GET_MODIFICATIONS_FAILED:
```

Comment:
INTERNAL SERVER ERROR for this looks weird to me, maybe change it to FORBIDDEN ?

Reply:
We throw  GET_MODIFICATIONS_FAILED exception when an error occurs during the execution of a request to the network modification server
and we do not know the nature of the problem